### PR TITLE
Update kubernetes-nmstate jobs

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
/hold
Let's see if rehearse is fine

File: github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml [hunk 1]
```diff
@@ -64,7 +61,7 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-k8s.sh"
-
+
```
Comment (rhrazdil) [inline github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml:64]: nit: extra whitespace

File: github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml [hunk 2]
```diff
@@ -96,6 +93,7 @@ presubmits:
               - "/usr/local/bin/runner.sh"
               - "/bin/sh"
               - "-c"
+              - "automation/check-patch.e2e-k8s.sh"
```
Comment (rhrazdil) [inline github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml:96]: Isn't this here by mistake?

Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml:96]: removing

File: github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml (hunk 1 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml]: Removing

**Comment by qinqon** [comment]
jobs look good
/override rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s 
/override rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s-future

**Pull request merged**